### ISSUE TITLE
Login without tournament

### ISSUE THREAD

**Issue by kevinhughes27** [kevinhughes27/tournament-app#207]
Currently you visit your tournament url and if you don't have a session you get redirected to login and your tournament id is stored in the session so we know where to redirect you after.

First issue: I need to guard `sign_in` on the presence of this id otherwise weird things would happen

2nd after #206 it would be nice to have a login without tournament which presents you a list of tournaments to login into.
